pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -638,124 +583,41 @@ func (k *Bootstrapper) restartControlPlane(cfg config.ClusterConfig) error {
 
 	client, err := k.client(hostname, port)
 	if err != nil {
-		return errors.Wrap(err, "getting k8s client")
+		klog.Infof("needs reconfigure: getting k8s client error: %v", err)
```

Comment:
Should this be an error?

Reply:
Good point ! Except it shouldn't be an error it should be a warning.
klog.Warning will not pollute the UI for the end user.

Reply:
Changed to `klog.Warningf`